### ISSUE TITLE
Windows command line showing invalid characters

### ISSUE THREAD

**Issue by brandonfarber** [stripe/stripe-cli#93]
### Issue
_Describe what happened and what you were trying to do_

I ran the stripe executable from the command line and while the connections seem to establish correctly, invalid characters are output to the console window which is both "ugly", but also tricky (i.e. to copy the webhook signing secret).

![image](https://user-images.githubusercontent.com/10439617/62963377-e090a800-bdce-11e9-957e-4d1c68997dc6.png)

### Expected Behavior
_Tell us what you expected to happen_

I expect the intention is to include these values in quotes, or to mark them up, however apparently the control characters aren't being recognized.

### Steps to reproduce
_What are the steps we can take to reproduce this and verify it's fixed?_

Windows 10 Home
Version	10.0.18362 Build 18362

I followed the very basic steps to download and execute the stripe.exe - nothing fancy. You can see the entire output of the commands I've run in the screenshot above.

Command line codepage is 437 with font Consolas if it matters (grabbed from command properties).

### Environment
_Select one of: macOS, Linux, Windows, Other_

Windows - see above for more details.

**Comment by tomer-stripe** [comment]
@brandonfarber sorry about that.

Could you try to run with `--color off` and see if that helps?

**Comment by ob-stripe** [comment]
Hi @brandonfarber, thanks for the report. Those weird characters are ANSI color codes, which unfortunately the Windows default console app does not support.

You can add the `--color=off` flag to disable the color codes. We also plan on adding support for setting this option permanently in your config file so you don't have to pass the flag every time.

Microsoft is also working on a new terminal app (https://github.com/microsoft/terminal) that should hopefully correctly handle ANSI color codes. You can also try using a third-party app like ConEmu or Cmder which should also correctly display colors.

**Comment by brandonfarber** [comment]
Thanks for the tip, I'll keep it in mind.

I don't know if you want to investigate ways to address automatically (maybe something along these lines: https://stackoverflow.com/questions/38044810/detect-ansi-compatible-console-from-windows-batch-file) or if the code is abstracted enough it's designed to be pretty much the same across all environments, but I figured I'd point out the issue. Feel free to close this since it's fairly minor, if the intention is to use the flag to switch off colors or wait on an updated console to be released.

**Comment by ob-stripe** [comment]
I think there may be things we can do to improve the experience on Windows here, so let's keep this open for now.

**Comment by ob-stripe** [comment]
Hey @brandonfarber, we just released v0.4.4. Colors should be displaying normally now (if you're using Windows 10 -- older Windows versions would still be affected).

Feel free to reply here or open a new issue if you notice anything amiss!

**Comment by brandonfarber** [comment]
Just tested and it seems ok. I'll let you know if anything comes up, thanks!
